assignment4.Rmd

```diff
@@ -114,3 +117,34 @@ biplot(pca)
 ```
```

Comment:
If you look at what the PC's are correlated with you can come up with an idea of what each of the 6 components are representing.